src/phase_ior_hard_write.c

```diff
@@ -19,7 +19,6 @@ static opt_ior_hard_write o;
 
 static ini_option_t option[] = {
   {"API", "The API to be used", 0, INI_STRING, NULL, & o.api},
-  {"posix.odirect", "Use ODirect", 0, INI_BOOL, NULL, & o.odirect},
```

Comment:
This seems like a change in the running of the benchmark?  It wasn't clear to me why this option was removed, or was it a broken option that didn't actually do anything?